lib/tasks/reverse_geocode.rake

```diff
@@ -0,0 +1,26 @@
+namespace :db do
+  desc "Reverse geocode each restroom's address (use db:reverse_geocode[dry_run] to preview changes)"
+  task :reverse_geocode, [:dry_run] => [:environment] do |t, args|
+    args.with_defaults(dry_run: false)
+
+    if args.dry_run
+      puts "Dry running reverse_geocode"
+    else
+      puts "Running reverse_geocode"
+    end
+
+    puts ""
+
+    Restroom.all.each do |restroom|
```

Comment:
Do we want to do this for _all_ Restrooms, or can we just do ones that have some columns missing?

Reply:
Depends if you're particularly fussy about the display, for example casing and `Street` vs `St.`